lms/views/admin/assignment.py

```diff
@@ -50,6 +50,7 @@ def assignment_dashboard(self):
             )
         )
 
+        assert assignment.course
```

Comment:
In the other PR I had a type ignore here but the assert is probably clearer.

assignment.course can be null for old assignments but you can't actually navigate to those in the admin pages, you can only access them via their course.

Reply:
Can you add this note as a comment in the code.